### PR TITLE
Feature/userprofile image

### PR DESCRIPTION
**Pull request by pmattmann**

### CODE REVIEW

**Comment by usu** [comment]
ng-file-upload and datepicker (css) missing in bower

**Comment by usu** [comment]
Birthday: today instead of null is displayed

**Comment by usu** [comment]
Big pictures not uploading

**Comment by usu** [comment]
Is saving even if I click on cancel

**Comment by usu** [comment]
How should we handle AutoFill of browsers? Dirty flag?

**Comment by pmattmann** [comment]
Corrected the following issues:
- added ng-file-upload to bower.json
- birthday displays null and can be set to null
- reset/cancel of dirty contols
- correct handling for autofill

issue about big pictures will be handled later... please merge accept anyway... thanks

File: module/EcampApi/src/EcampApi/Resource/Image/ImageResourceListener.php [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    {
+        $this->listeners[] = $events->attach('fetch', array($this, 'onFetch'));
+        //$this->listeners[] = $events->attach('fetchAll', array($this, 'onFetchAll'));
+    }
+
+    public function onFetch(ResourceEvent $e)
+    {
+        $id = $e->getParam('id');
+
+        var_dump($id);
```
Comment (usu) [inline module/EcampApi/src/EcampApi/Resource/Image/ImageResourceListener.php:30]: please remove

**Pull request merged**